ui/org.eclipse.pde.ui/src/org/eclipse/pde/internal/ui/pderesources.properties

```diff
@@ -488,6 +488,7 @@ SchemaAttributeDetails_use=Use:
 AbstractPluginBlock_counter={0} out of {1} selected
 AbstractTargetPage_setTarget=Set as Active Target Platform
 AbstractTargetPage_reloadTarget=Reload Target Platform
+AbstractTargetPage_openPreferences=Show all Target Platforms known to the IDE
```

Comment:
@merks, @laeubi what do you think about
```suggestion
Open 'Target Platform' preference page to manage all targets known to the IDE.
```
or just `Open 'Target Platform' preference page to manage all known targets.` or a bit more _technical_ `Open 'Target Platform' preference page to manage all available targets.`?

I think this should cover both aspects, mentioning that the button opens the preference page and that this page lists all known targets.

Reply:
I think we should not use `'` in the text there is no need for it.

"all available targets" don't sound right for me, because there could be numerous targets that we can't show here e.g. I don't think targets are listed if one D&D a file into the IDE (but I haven't specifically tested that).

Reply:
I would try to make the sentence grammatically complete with the word 'the' added:
```
Open the 'Target Platform' preference page to manage all known targets
```
Note that there is precedent for including the quotes

![image](https://github.com/eclipse-pde/eclipse.pde/assets/208716/057a3e5e-1a74-4e9d-97a4-0122b2e08468)

Reply:
> Note that there is precedent for including the quotes

At this place it even looks more strange (to me) because they are already highlighted by a link, but at least one is already at the preferences and there is a direct connection to `'thing'` we don't have here, so maybe even don't focus at the "page" aspect at all and simply name it 

`Open the target platform preference`

Reply:
I'm fine with the latter one as it is basically saying what is done by the action. The implication that the preference contains all known target definitions is given once the preference page is shown.

Reply:
The intention of the single-quotes was to emphazie that `Target Platform` is the title of the preference. If the tool-tip would support formatting like here in GitHub I would suggest a text like:
Open the `Target Platform` preference page...

So the single-quotes were the fall-back. But I'm also fine with removing them, I would just write Target Platform in capitalized like it is done in the page title.
Regarding if it is all known/available/... targets: That page definitively lists all target files in any open project, I don't know if it shows targets dropped to the IDE, but also don't know if that's even possible. We could just remove that part or just be vague and say `all targets`.

Based on that my suggestions are either
`Open the Target Platform preference`
or
`Open the Target Platform preference to manage all targets.`

Reply:
Given it's Window -> **Preferences**, and the title of the dialog is **Preferences** is think the word preference should be plural or probably better would be preference page like here:

![image](https://github.com/eclipse-pde/eclipse.pde/assets/208716/094d0b38-05b0-4bf7-983e-cb7abb847b8d)

Reply:
I would prefer not to include "page"  or any "manage XYZ" because maybe in the future we want to show multiple pages and "PreferencePage" is really a technical term, to open the Preferences we also have "Window > Preferences" and not "Window > Workspace Preferences Dialog"

That's why I think even `Open preferences` is enough as it is clear from the context...

Reply:
I did show reference to existing terminology that words it exactly this way.  There is much to be said for consistency...

I would wonder if Open preference would open the preferences for the editor.

So maybe is okay and still short enough?
```
Open the Target Platform preferences
```

Let's hope we don't discuss colors next, then we'll never find agreement.  😝

Reply:
> I did show reference to existing terminology that words it exactly this way. There is much to be said for consistency...

We can also adjust the references, there is so much things in platform that are far from optimal, if we always try to be "consistent" we would never be able to improve anything. e.g there are a lot of Preference pages that do not react well to size changes, should we (for consistency) make all pages react poorly to size changes?

Beside that, as mentioned this is *inside* the preferences dialog, while here we are inside an Editor and the other Icons simply read `Export` (not `Open Export Dialog (Wizard)`) and "`Help` (not `Open Eclipse Help (Dialog)`) so if we want to be really consistent the new icon should simply read `Preferences` ...

Reply:
Export is a verb.   👅   Preferences... is appropriate for a menu, not for a tool tip...

Yes, consistently bad is not good.  But I don't think what I showed is bad; it's proper well-worded English.

I suggest doing a search `preference page` in `*.properties` and finding all consistently similar examples:

![image](https://github.com/eclipse-pde/eclipse.pde/assets/208716/e86df7fe-8325-4a02-a09e-42fb7b55a2cb)

If we determine they are all consistently bad, we should consistently fix them all, rather than do something different.  I don't think being inconsistent is ever good.

In any case, I have said **more than enough**.  Hannes, as team lead can decide whatever he deems best and I have provided sufficient input to the process.  👼

Reply:
> I like the simplistic approach of `Open the Target Platform preferences` the best.

Agree. Lets continue with that. I think that's a good compromise.